### PR TITLE
Feature changes pending

### PR DESCRIPTION
**Pull request by nicolasnunez-eb**
When a supplier make a change on the taxpayer, then the taxpayer change his status to "changes pending"

### CODE REVIEW

File: supplier_app/views.py [hunk 1]
```diff
@@ -49,7 +50,9 @@
 )
 from supplier_app.exceptions.taxpayer_exceptions import (
```
Comment (ricardocaif-eb) [inline supplier_app/views.py]: nit: import order

Comment (nicolasnunez-eb) [reply supplier_app/views.py]: done

File: supplier_app/tests/test_view.py [hunk 2]
```diff
@@ -1942,6 +1941,45 @@ def test_approve_without_workday_id_set_shows_msg_error(self):
 
         self.assertContains(response, TAXPAYER_WITHOUT_WORKDAY_ID_MESSAGE)
 
+    def test_taxpayer_approve_with_existent_workday_id_should_remain_taxpayer_status(self):
```
[the remark this answers is not in the document]
Comment (nicolasnunez-eb) [reply supplier_app/tests/test_view.py]: done

File: supplier_app/models.py [hunk 3]
```diff
@@ -19,12 +19,14 @@
     BANK_ACCOUNT_ALLOWED_FILE_EXTENSIONS,
     PAYMENT_TERMS,
     PAYMENT_TYPES,
-    TAXPAYER_STATUS,
     TAXPAYER_CERTIFICATE_MAX_SIZE_FILE,
     TAXPAYER_ALLOWED_FILE_EXTENSIONS,
-    get_taxpayer_status_choices,
 )
 
+from supplier_app.constants.taxpayer_status import (
```
[the remark this answers is not in the document]
Comment (nicolasnunez-eb) [reply supplier_app/models.py]: done

**Pull request merged**